/**:
  ros__parameters:
    cluster_size_min : 20
    cluster_size_max : 10000
    max_cluster_distance : 1.0
    height_filter_thres : 0.2
